logproof/src/fields.rs

```diff
@@ -48,10 +48,15 @@ where
  *
  * SEAL uses Q =
  * 0x7fffffd8001 * 0x7fffffc8001 * 0xfffffffc001 * 0xffffff6c001 * 0xfffffebc001
+ *
+ * This can be derived by running
+ * `CoefficientModulus::bfv_default(8192, SecurityLevel::TC128)`
+ * or by running the underlying SEAL function
+ * `CoeffModulus::BFVDefault(8192, sec_level_type::tc128)`
  */
 #[derive(MontConfig)]
-#[modulus = "421249101157150430150591791601812858371395928330411389778873040897"]
 #[generator = "3"]
+#[modulus = "421249101157150430150591791601812858371395928330411389778873040897"]
```

Comment:
Hmm... do we use the last modulus or not without the special modulus? You can take a ciphertext and count that you should have 2 polynomials, each with 8192*{4 | 5} RNS lanes. That will tell you whether this is the right modulus or not.

Reply:
I think we do not, since there are 4 RNS lanes in the data. The last modulus is not used in the data itself. So I think we will need to change this number, but I have been focusing on the case where the RNS lanes is 1 so that I can ensure that the full knowledge proof is correct.